nixos/modules/services/hardware/throttled.nix

```diff
@@ -8,6 +8,12 @@ in {
   options = {
     services.throttled = {
       enable = mkEnableOption "fix for Intel CPU throttling";
+
+      extraConfig = mkOption {
+        type = types.str;
+        default = "";
```

Comment:
An example would be nice

Reply:
This could become quite long. I believe the repos docs are just fine for such an experimental project.